src/styles/card-review.css

```
@@ -0,0 +1,30 @@
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__review-stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("../images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.card__review-stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url("../images/star-active.svg");
```

```suggestion
.card__review-stars--1 .stars__star:nth-child(-n+1)
.card__review-stars--2 .stars__star:nth-child(-n+2)
.card__review-stars--3 .stars__star:nth-child(-n+3)
.card__review-stars--4 .stars__star:nth-child(-n+4)
.card__review-stars--5 .stars__star:nth-child(-n+5) {
  background-image: url("../images/star-active.svg");
```

Для того, чтоб если ты в классе вместо 4 поставишь 2 то у тебя на карточке было 2 звезды